sycl/test/check_device_code/cuda/matrix/matrix-nvptx-tf32-test.cpp

```diff
@@ -79,15 +79,6 @@ int main() {
               sg, sub_c, accC.template get_multi_ptr<access::decorated::yes>(),
               N, layout::row_major);
 
-          // Round a, b to tf32
```

Comment:
I think this file/directory needs an entry in CODEOWNERS file. Please merge a separate PR with that change. Once done, I expect not review from SYCL RT team will be required here.

Reply:
I've done this here: https://github.com/intel/llvm/pull/11582